FCModel/FCModel.m

```diff
@@ -330,7 +330,7 @@ + (id)cachedObjectWithIdentifier:(id)identifier ignoreFieldsForInvalidation:(NSS
 
 + (NSError *)executeUpdateQuery:(NSString *)query, ...
 {
-    checkForOpenDatabaseFatal(YES);
+    if (! checkForOpenDatabaseFatal(NO)) return nil;
```

Comment:
Wouldn't we want to return an `NSError` here that says the database is closed, instead of nil?

Same thing here: https://github.com/bryanhales-wf/FCModel/pull/1/files#diff-12f4ae395dbee2c68fb9fa224d60d640R353